docs_user/modules/proc_decommissioning-rhosp-standalone-ceph-NFS-service.adoc

```diff
@@ -2,31 +2,20 @@
 
 = Decommissioning the {rhos_prev_long} standalone Ceph NFS service
 
-If the deployment uses CephFS through NFS, you must inform your {rhos_prev_long}({OpenStackShort}) users
-that the old, standalone NFS service will be decommissioned. Users can discover
-the new export locations for their pre-existing shares by querying the {rhos_component_storage_file} API.
-To stop using the old NFS server, they need to unmount and remount their
-shared file systems on each client. If users are consuming the {rhos_component_storage_file} shares via
-the {rhos_component_storage_file} CSI plugin for {OpenShift}, this migration can be done by scaling down
-the application pods and scaling them back up. Clients spawning new workloads
-must be discouraged from using share exports via the old NFS service. The {rhos_component_storage_file}
-will no longer communicate with the old NFS service, and so it cannot apply or
-alter any export rules on the old NFS service.
+If your deployment uses CephFS through NFS, you must decommission the {rhos_prev_long}({OpenStackShort}) standalone NFS service. Since future software upgrades do not support the previous NFS service, it is recommended that the decommissioning period is short.
 
-Since the old NFS service will no longer be supported by future software
-upgrades, it is recommended that the decommissioning period is short.
+You can discover the new export locations for your existing shares by querying the Shared File Systems API. To stop using the previous NFS server, you need to unmount and remount the shared file systems on each client. If you are consuming the {rhos_component_storage_file} shares with the {rhos_component_storage_file} CSI plugin for {OpenShift}, you can migrate the shares by scaling down the application pods and scaling them back up. Clients spawning new workloads should not use share exports through the previous NFS service. The {rhos_component_storage_file} no longer communicates with the previous NFS service, and so it cannot apply or alter any export rules on the previous NFS service.
```

Comment:
What is the intent of the paragraph that starts with “you can discover”? Are some of these things that customers need to do before they start the decommissioning or are some of them suggestions? E.g.,
* Discover the new export locations for your existing shares by querying the Shared File Systems API.
* To stop using the previous NFS server, you need to unmount and remount the shared file systems on each client.
* If you are consuming the {rhos_component_storage_file} shares with the {rhos_component_storage_file} CSI plugin for {OpenShift}, you can migrate the shares by scaling down the application pods and scaling them back up.

Reply:
Yes, it helps. I think I will put these bullet points under a "Prerequisites" heading, and format them as a list for easier readability.